distribution/config/index.js

```diff
@@ -15,32 +12,40 @@ var API = exports.API = Object.freeze({
   CHUNKS_PER_REQUEST: 10
 });
 
-var IOTA_API = exports.IOTA_API = Object.freeze({
+var IOTA_API = Object.freeze({
   PROVIDER_A: "http://18.188.17.130:14265",
   PROVIDER_B: "http://18.220.1.63:14265",
   PROVIDER_C: "http://18.216.90.80:14265",
   ADDRESS_LENGTH: 81,
   MESSAGE_LENGTH: 2187
 });
 
-var UPLOAD_STATUSES = exports.UPLOAD_STATUSES = Object.freeze({
+var UPLOAD_STATUSES = Object.freeze({
   PENDING: "PENDING",
   SENT: "SENT",
   FAILED: "FAILED"
 });
 
-var DOWNLOAD_STATUSES = exports.DOWNLOAD_STATUSES = Object.freeze({
+var DOWNLOAD_STATUSES = Object.freeze({
   STANDBY: "STANDBY",
   PENDING: "PENDING",
   RECEIVED: "RECEIVED",
   FAILED: "FAILED"
 });
 
-var FILE = exports.FILE = Object.freeze({
+var FILE = Object.freeze({
   MAX_FILE_SIZE: 200 * 1000,
   CHUNKS_PER_SECTOR: 1000000,
   CHUNK_TYPES: {
     METADATA: "METADATA",
     FILE_CONTENTS: "FILE_CONTENTS"
   }
-});
\ No newline at end of file
+});
+
+module.exports = {
+  API: API,
```

Comment:
`{ API, DOWNLOAD_STATUSES, ... }` is shorthand for `{ API: API, DOWNLOAD_STATUSES: DOWNLOAD_STATUSES, ... }`

Reply:
This was automatically generated.